exercises/practice/hello-world/.meta/proof.ci.js

```diff
@@ -0,0 +1 @@
+export const hello = () => 'Hello, World!';
```

Comment:
I don't really understand why the proof uses JavaScript. The goal of the proof is to show that there is a solution that can be written by the student that passes all the tests. If I look at the [stub implementation file](exercises/practice/hello-world/hello-world.wat), I would have expected to just have a `proof.ci.wat` file that is identical to the stub file, but with the correct text.